docs/reference/ccr/apis/auto-follow/resume-auto-follow-pattern.asciidoc

```diff
@@ -0,0 +1,88 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-resume-auto-follow-pattern]]
+=== Resume auto-follow pattern API
+++++
+<titleabbrev>Resume auto-follow pattern</titleabbrev>
+++++
+
+Resumes an auto-follow pattern.
+
+[[ccr-resume-auto-follow-pattern-request]]
+==== {api-request-title}
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT /_ccr/auto_follow/my_auto_follow_pattern
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index_patterns" :
+  [
+    "leader_index"
+  ],
+  "follow_index_pattern" : "{{leader_index}}-follower"
+}
+--------------------------------------------------
+// TEST[setup:remote_cluster]
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+POST /_ccr/auto_follow/my_auto_follow_pattern/pause
+--------------------------------------------------
+// TEST
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+POST /_ccr/auto_follow/<auto_follow_pattern_name>/resume
+--------------------------------------------------
+// TEST[s/<auto_follow_pattern_name>/my_auto_follow_pattern/]
```

Comment:
We typically use literals here:
`POST /_ccr/auto_follow/<auto_follow_pattern_name>/resume`

Reply:
I made it like this to mirror what is done in docs/reference/ccr/apis/follow/post-pause-follow.asciidoc but maybe it's wrong there too?

Reply:
The format here is based on the API reference template:
https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L39

It's likely that `docs/reference/ccr/apis/follow/post-pause-follow.asciidoc` followed a previous iteration of that template.

If you'd like to leave this as-is for consistency, I think that's okay.  I don't feel strongly about it.

Reply:
Thanks, I forgot about this template. I adapted the doc to follow the template and not add more debt.